src/courtroom.cpp

```diff
@@ -2830,9 +2833,11 @@ void Courtroom::chat_tick()
     ui_vp_chat_arrow->play(
         "chat_arrow", f_char,
         f_custom_theme); // Chat stopped being processed, indicate that.
-    additive_previous =
-        additive_previous +
-        filter_ic_text(f_message, true, -1, m_chatmessage[TEXT_COLOR].toInt());
+    QString f_message_filtered = filter_ic_text(f_message, true, -1, m_chatmessage[TEXT_COLOR].toInt());
+    for (int c = 0; c < max_colors; ++c) {
```

Comment:
Oh, this starts from c=0? So it's different from courtroom_fonts.ini thing?

Reply:
ui_vp_message uses the c0 as defined by config.ini since there are characters which, for example, use black instead of white for their message text.